sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionPumpManager.java

```diff
@@ -244,8 +264,13 @@ void startPartitionPump(PartitionOwnership claimedOwnership, Checkpoint checkpoi
                 });
 
             if (maxWaitTime != null) {
-                partitionEventFlux = receiver
-                    .windowTimeout(maxBatchSize, maxWaitTime);
+                if (HAS_BACKPRESSURE_WINDOW_TIMEOUT) {
+                    partitionEventFlux = receiver
+                        .windowTimeout(maxBatchSize, maxWaitTime, true);
```

Comment:
This is unfortunate, but looks like Reactor added this as an ambiguous call site as there was a previous `windowTimeout(int, Duration, Scheduler)` to `true` will need to be cast as a boolean

Reply:
Ah, I see. hmm.. the new operator takes primitive-boolean, not reference-Boolean; wondering why compiler tries to resolve primitive-boolean to reference-scheduler. 

Given its compile time error, would it be possible that the classpath somehow resolved to an old reactor core (< 3.4.19)?